.env

```diff
@@ -43,4 +43,5 @@ DATABASE_URL="postgresql://symfony:ChangeMe@127.0.0.1:5432/app?serverVersion=13&
 ###< symfony/mailer ###
 
 PROGRAMME_MIN_TIME_IN_MINUTES=15
-PROGRAMME_MAX_TIME_IN_MINUTES=360
\ No newline at end of file
+PROGRAMME_MAX_TIME_IN_MINUTES=360
+MAX_PER_PAGE=10
```

Comment:
This is a bit too vague. If you plan to make this the default of all the resources in your API then it's fine. However, you might get into a situation where you need different page size values for different resources and a more specific name will help. This is not an issue, just a heads-up.

Reply:
I wanted to make this the default.